src/chains.jl

```diff
@@ -76,9 +77,21 @@ function Chains(
                     chain = 1:size(val, 3))
 
     # Create the new chain.
-    return Chains(arr, evidence, _name_map, info)
+    return Chains(arr, evidence, _name_map, info, weights)
 end
 
+function Chains(
+    value::AxisArray{<:Union{Missing,Real},3}, 
+    logevidence::Union{Missing,Real}, 
+    name_map::NamedTuple, 
+    info::NamedTuple
+)
+    return Chains(
+        value, logevidence, name_map, info, StatsBase.UnitWeights{Bool}(size(value, 1))
+    )
+end
+
+
```

Comment:
I don't think this is needed. More constructors without clear benefit mainly creates confusion about what is the "official" way to construct it. In general, IMO keyword arguments are clearer.
```suggestion
```

Reply:
This just defaults to unit weights, to avoid breaking any old code that creates `Chains` without including weights. If I don't include it, some of the tests break.

Reply:
The PR is breaking anyway. IMO it is much better to fix the tests than to add an additional constructor.